### PR TITLE
Fixed focus indicator on the schedule veteran page

### PR DESCRIPTION
**Pull request by sahalliburton**
Resolves [CASEFLOW-1571](https://vajira.max.gov/browse/CASEFLOW-1571)

### Description

Fixes the cutoff focus indicator when tabbing between Hearing Day selections on the Schedule Veteran page

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] Visible focus indicator on docket picker is not covered by other elements

### Testing Plan

1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the hearing schedule: http://localhost:3000/hearings/schedule
3. Click the “Schedule Veteran” button at the top left
4. Select the St. Petersburg, FL RO from the dropdown
5. Tab to the first hearing day and make sure that the focus indicator is fully visible

### User Facing Changes
 - [ ] Screenshots of UI changes added to PR & Original Issue

 BEFORE|AFTER
 ---|---
Screen Shot 2021-06-28 at 9 50 26 AM|HearingDayNavFocusIndicitorAfter
Screen Shot 2021-06-28 at 11 52 46 AM|Screen Shot 2021-06-28 at 11 52 30 AM

### CODE REVIEW

**Comment by nicholer** [comment]
Looks good to me!

**Comment by charleyf** [comment]
When a button that's not the 1st one for a month is selected, there's a space between the highlight and the line above.
TopSpacing

Maybe something like this and a CSS rule targeting the first of each month?
```css
  li {
    margin-top: -20px;
    padding: 10px;
  }
```

File: client/app/styles/hearings/_hearings.scss [hunk 1]
```diff
@@ -366,6 +366,15 @@ $form-width: 575px;
...
     background-color: $color-white;
+    margin-bottom: 5px;
+  }
+
+  li {
+    padding-left: 5px;
+
+    button:focus {
+      outline-offset: 0px;
+    }
```
[the remark this answers is not in the document]
Comment (sahalliburton) [reply client/app/styles/hearings/_hearings.scss]: Thanks @charleyf just applied the fix!

**Review by charleyf — APPROVED** [review]
Border appears for all elements and isn't cut off. Looks good!

**Pull request merged**